django/db/backends/oracle/introspection.py

```diff
@@ -59,7 +59,10 @@ def get_table_list(self, cursor):
 
     def get_table_description(self, cursor, table_name):
         "Returns a description of the table, with the DB-API cursor.description interface."
+        # disable statement caching incase tables or columns have been altered
+        self.connection.set_statement_cache_size(0)
```

Comment:
Why are we not just using `connection.clear_old_columns()` here?
If this is really needed, I'd consider replacing `clear_old_columns()` with a context manager:
```py
with connection.disabled_stmt_cache():
   cursor.execute(...)
```